FreeRTOS/Demo/RX600_RX64M_RSK_Renesas_e2studio/makefile.init

```diff
@@ -2,7 +2,7 @@
 # Automatically-generated file. Do not edit!
 ################################################################################
 
-export INC_RX=C:\devtools\Renesas\RX\2_1_0\include
-export RXC_LIB=C:\devtools\Renesas\RX\2_1_0\bin
-export BIN_RX=C:\devtools\Renesas\RX\2_1_0\bin
-PATH := $(PATH):C:\devtools\Renesas\RX\2_1_0\bin
\ No newline at end of file
+export INC_RX=C:\PROGRA~2\Renesas\RX\2_8_1\include
```

Comment:
This looks kind of mangled, compared to Program Files (x86) which was used above

Reply:
The file is generated by IDE/toolchain. IMHO this file should not have been added to Git in the first place.
It should get overwritten by user's IDE when they import and try to build

Reply:
In case of the latest e2 studio v7.8.0(32bit version) and v2020-07(64bit version), the following files are not necessary. (These files are no longer created when you create new proect. Only very old versions of the e2 studio created/used these files.)

(1) .HardwareDebuglinker
(2) .info
(3) makefile.init